### PR TITLE
Fixes #14223 - Handles atomic machine detection properly

### PR DESCRIPTION
**Pull request by parthaa**
Made the code check for "Atomic Host" in /etc/os-release

Works great for redhat 7.2

### CODE REVIEW

**Comment by parthaa** [comment]
Look at https://bugzilla.redhat.com/show_bug.cgi?id=1200122 for more info

File: templates/rhsm-katello-reconfigure.erb [hunk 1]
```diff
@@ -23,8 +23,9 @@ chmod 644 $KATELLO_CERT_DIR/$KATELLO_SERVER_CA_CERT
 echo "$KATELLO_DEFAULT_CA_DATA" > $KATELLO_CERT_DIR/$KATELLO_DEFAULT_CA_CERT
 chmod 644 $KATELLO_CERT_DIR/$KATELLO_DEFAULT_CA_CERT
 
-# if not atomic machine, regular rhel then handle traditional way
-if [ -e "/run/ostree-booted" ]
+# if atomic machine handlle it the atomic way, else handle the regular rhel way
```
Comment (stbenjam) [inline templates/rhsm-katello-reconfigure.erb]: handle*

**Comment by jlsherrill** [comment]
ACK

**Comment by jlsherrill** [comment]
thanks @parthaa

**Pull request merged**